Julia/cp/subsetsOfArray.jl

```diff
@@ -0,0 +1,66 @@
+#= For any given array there are 2^N subsets. Thus generating each and every subset using BitMasking.
+We follow the binary representation of whole numbers and use them as boolean value if a character in the Set
+will occur or not. We will remove the duplicate subsets and display the non null unique subsets.
+=#
+
+## Function 
```

Comment:
Using multiline comments

Reply:
Could you explain what did you meant by adding repl.it link